README.md

```diff
@@ -242,6 +242,33 @@ const client = new faunadb.Client({
 })
 ```
 
+## Known issues
+
+### Using with Cloudflare Workers
+
+Cloudflare Workers have neither XMLHttpRequest nor fetch in the global scope.
+Therefore, the `cross-fetch` package is unable to inject its own `fetch()` function, and throws an error.
+The `fetch()` function is injected via a closure, so the workaround would be to pass
+the fetch objects when initiating the FaunaDB client config. Cloudflare Workers also
+doesn't support the use of an AbortController, which terminates requests as well as streams.
+Here is a workaround:
+
+```javascript
+const c = new faunadb.Client({
+  secret: 'your secret',
+  fetch: (url, params) => {
+    const signal = params.signal
+    delete params.signal
```

Comment:
why `delete` is necessary here?

Reply:
Cloudflare doesn't implement signal into their fetch, if the signal property is not removed, an error would be thrown